src/applications/vaos/tests/new-express-care-request/form-submission.unit.spec.jsx

```diff
@@ -157,13 +157,15 @@ describe('VAOS integration: Express Care form submission', () => {
       },
     );
 
-    fireEvent.change(await screen.getByLabelText(/tell us about your/i), {
+    await screen.findByText(/tell us about your cough/i);
```

Comment:
I think this will fix one of the flaky EC tests. We need to wait for text box and symptom specific heading to load before selecting info.